shared/message/vision/VisualMesh.proto

```diff
@@ -24,6 +24,9 @@ package message.vision;
 import "google/protobuf/timestamp.proto";
 import "Matrix.proto";
 
+/**
+ * Contains the details of a Visual Mesh™ measurement.
+ */
 message VisualMesh {
     message Row {
         float phi      = 1;
```

Comment:
These didn't get documented!

Reply:
These seem like something you'd be more qualified to write

Reply:
I had no idea about most of it. I have added comments to the ones I know about, but I have very little idea about the rest.

Reply:
Rows are sort of like the concentric rings of the mesh spreading out from the robot (sort of because they aren't really concentric). `phi` is the angle out from the camera. `segments` are the (on-screen?) nodes in the row.